retinaface/RetinaFace.py

```diff
@@ -245,16 +245,20 @@ def extract_faces(
 
         x = facial_area[0]
         y = facial_area[1]
-        w = facial_area[2]
-        h = facial_area[3]
+        w = facial_area[2] - x
+        h = facial_area[3] - y
```

Comment:
Is this a typo? I don't understand why that would be minus, and re-implementing with + instead of - works on my environment.

Reply:
Because 3rd and 4th points are x and y coordinates of right down area. We can find width and height by subtracting them to left up area coordinates.

Reply:
If you use plus, then you will have greater facial area. But it may exceeds the image coordinates and throw an exception. Correct calculation should be minus.

Reply:
Okay, thanks for the clarification!